spec/models/activerecord_obstacle_course_spec.rb

```diff
@@ -274,12 +276,12 @@
 
   it "returns all orders for item_4" do
     # ------------------ Inefficient Solution -------------------
-    order_ids = OrderItem.where(item_id: item_4.id).map(&:order_id)
-    orders = order_ids.map { |id| Order.find(id) }
+    # order_ids = OrderItem.where(item_id: item_4.id).map(&:order_id)
+    # orders = order_ids.map { |id| Order.find(id) }
     # -----------------------------------------------------------
 
     # ------------------ Improved Solution ----------------------
-    #  Solution goes here
+      orders = item_4.orders
```

Comment:
@s-espinosa @carmer would this be possible without the order-item table? Does the existence of a joins table automatically imply a many:many relationship?